### ISSUE TITLE
Unable to remove gocd authentication

### ISSUE THREAD

**Issue by srk12318** [gocd/gocd#10540]
##### Issue Type

- Bug Report

##### Summary

Unable to remove gocd authentication

###### Basic environment details

* Go Version: `Go Version: 18.2.0 `
* JAVA Version: `java 1.8`
* OS: ` OEL 7 `
* Browser vendor and version (if relevant): `MS EDGE` M

###### Additional Environment Details

hello,

i'm unable to login into gocd console as some issue with ldap configuration. so I have removed security key in cruise-config.xml to remove authentication and restarted gocd server . but still coming login page asking username and password. is there any way to remove authentication for gocd.

error message on login screen
There was an error authenticating you. Please check the go server logs, or contact the go server administrator

I dont see any errors ingoserver logs

below is ldap plugin log

2022-06-22 06:31:12,507 INFO  [Thread-76] LdapPlugin:72 - Loading plugin cd.go.authentication.ldap version 1.0.1-49
2022-06-22 06:31:12,579 WARN  [Thread-76] LdapPlugin:97 - Request go.plugin-settings.get-configuration is not supported by plugin.

**Comment by chadlwilson** [comment]
As this is an over 4 year old version of GoCD, it might be difficult to give specific support if some aspects have changed since then.

Can you be more specific about what you changed in `cruise-config.xml`? Did you remove the entire `<security>...</security>` configuration or did you just remove the `<authConfig>`s ?

Might want to double-check you are editing the correct configuration file and that your edits are being reflected.

**Comment by srk12318** [comment]
Thanks for your prompt response. i have removed <security>...</security> entire from cruise-config.xml but didn't work.
however we identified the root cause of login issue that something dynatrace oneagent(recently installed) causing the issue, when we stopped the oneagent, we are able to login now. we will raise this issue with dynatrace.  if below issue not related to gocd anymore. i will close this. Once again thanks

go-server logs
0cf9596aa - java.lang.ClassCastException: com.dynatrace.agent.introspection.uem.impl.CacheHookingRequestWrapper cannot be cast to org.eclipse.jetty.server.Request

2022-06-22 07:55:11,849 ERROR [*************] BasicAuthenticationFilter:64 dt.trace_sampled: true, dt.trace_id: 78336a7a0d29fbad5875c1e386b00326, dt.span_id: 3d4a36996ebc2172 - java.lang.ClassCastException: com.dynatrace.agent.introspection.uem.impl.CacheHookingRequestWrapper cannot be cast to org.eclipse.jetty.server.Request
